### ISSUE TITLE
error when launching single dispatcher using --hunger-policy

### ISSUE THREAD

**Issue by maikia** [paris-saclay-cds/ramp-board#426]
When (locally) I follow the docs [to run an event](https://paris-saclay-cds.github.io/ramp-docs/ramp-board/stable/create_ramp_event.html#launch-several-dispatchers-at-once) and run a single dispatcher using a given command: 

`~/ramp_deployment $ ramp launch dispatcher --event-config events/iris_test/config.yml --hunger-policy sleep -vv`

I am receiving an error:

> Error: no such option: --hunger-policy

It runs alright if the --hunger-policy option is removed

**Comment by glemaitre** [comment]
True. We need to add the parameter to the signature. The code already use a default: https://github.com/paris-saclay-cds/ramp-board/blob/master/ramp-engine/ramp_engine/cli.py#L81
